core/src/main/java/org/apache/iceberg/RemoveSnapshots.java

```diff
@@ -79,10 +80,11 @@ public void accept(String file) {
   private TableMetadata base;
   private long expireOlderThan;
   private int minNumSnapshots;
+  private ExpireSnapshotResult expireSnapshotResult;
   private Consumer<String> deleteFunc = defaultDelete;
   private ExecutorService deleteExecutorService = DEFAULT_DELETE_EXECUTOR_SERVICE;
 
-  RemoveSnapshots(TableOperations ops) {
+  public RemoveSnapshots(TableOperations ops) {
```

Comment:
This is created by calling `expireSnapshots` on a table. Why did you make it public?